pkgs/top-level/all-packages.nix

```diff
@@ -4239,13 +4239,7 @@ with pkgs;
 
   cpcfs = callPackage ../tools/filesystems/cpcfs { };
 
-  # coreutils 9 messes up some binary format on darwin
-  # https://github.com/NixOS/nixpkgs/pull/141684
-  # test by trying to build alacritty for example
```

Comment:
Anyone happen to have an example which doesn't have quite as massive dependency/rebuild footprint for testing?

Reply:
r-burns was saying `jx` was the fastest one for him.
My machine is quite old, it still took me 6 hours I think.
If you leave it overnight, it should be fine.
I made sure by also testing `alacritty`.

Reply:
No, my jx testing was all on linux. I didn't encounter any issues on darwin.